tests/integration/aof.tcl

```diff
@@ -253,13 +253,13 @@ tags {"aof external:skip"} {
     ## Test that the server exits when the AOF contains a unknown command
     create_aof $aof_dirpath $aof_file {
         append_to_aof [formatCommand set foo hello]
-        append_to_aof [formatCommand bla foo hello]
+        append_to_aof [formatCommand blah foo hello]
```

Comment:
as mentioned earlier, I'm changing all `bla` to `blah`.

If the code is trying to say "this thing is only a 3 letter item", then I'd suggest using something more like `xyz`, whereas I think here it's trying to mean "blah".